src/test/java/com/radixdlt/crypto/SignaturesTest.java

```diff
@@ -21,14 +26,85 @@ public void verify_that_default_signature_scheme_is_ecdsa() {
         assertEquals(nonEmptySignatures.keyToSignatures().size(), 1);
     }
 
+    @Test(expected = Test.None.class /* no exception expected */)
```

Comment:
What is this for? If we don't expect an exception (which is the default), then there is no need for this

Reply:
Bah, you got to see some outdated code there, hmm strange, did your browser not refresh? Fixed anyway..